modules/rosa-rn-storage.adoc

```diff
@@ -0,0 +1,26 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// rosa_release_notes/rosa-rn-new-features.adoc
+
+[id="rosa-rn-storage_{context}"]
+= Storage
+
+[id="rosa-rn-encrypted-os-node_{context}"]
+== Encrypted-at-rest OS/Node storage
+
+Red Hat master nodes use encrypted-at-rest-EBS storage. Customers do not have access to the Red Hat master nodes, which are used for managing the {product-title} (ROSA) services.
+
+[id="rosa-rn-encrypted-pv_{context}"]
+== Encrypted-at-rest PV
+
+By default, EBS volumes used for persistent volumes (PVs) are encrypted-at-rest.
+
+[id="rosa-rn-block-storage_{context}"]
+== Block storage (RWO)
+
+PVs are backed by Amazon Web Services (AWS) block storage, known as AWS EBS, that is Read-Write-Once.
```

Comment:
@wgordon17 - Although not fully supported do we want to include here the ability to use EFS and how one would do so?   Someone looking for storage docs might want to do RWX as well...

Reply:
If we _were_ to mention it, we'd probably want to update the KCS article to make it more generalized toward both OSD _and_ ROSA https://access.redhat.com/articles/5025181

Reply:
@wgordon17  @okashi18  Please let me know if you're planning to update that PCS article and whether you want to include a general comment about the ability to use EFS. Thanks.